### PR TITLE
updated project to kotlin 1.9.22 and build config to 5.3.5

### PR DESCRIPTION
**Pull request by demonar**
Updates:

- Kotlin 1.9.22
- buildconfig 5.3.5
- java 11

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -55,3 +55,10 @@ allprojects {
         mavenCentral()
     }
 }
+
+tasks.register("clean", Delete::class) {
```
Comment (arkivanov) [inline build.gradle.kts:59]: Could you please explain wehy do we need this? I used to have a `clean` task already.

**Review by arkivanov — COMMENTED** [review]
Thank you for taking care of this! There is no CI setup at the moment for PRs. Could you please make sure you have run all tests from the `tests` folder? Just run the following.

```
cd tests
./gradlew build
```

**Comment by demonar** [comment]
No worries 😄, I did run the tests before submitting the PR. All good.

**Comment by arkivanov** [comment]
Thanks for running the tests! Could you please also explain why do we need the `clean` task? There used to be a `clean` task already defined.

**Comment by demonar** [comment]
This one cleans all the modules build folders.

**Review by arkivanov — APPROVED** [review]

**Comment by arkivanov** [comment]
Thank you! The [new version](https://github.com/arkivanov/parcelize-darwin/releases/tag/0.2.4) has been published and should become available soon.

**Comment by demonar** [comment]
Glad I could help 😄

**Pull request merged**